starter-code/index.js

```diff
@@ -1,11 +1,41 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
+
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort((a, b) => a - b);
+  }
+
+  get(pos) {
+    this.length = this.items.length;
+    if (pos > 0) { return this.items[pos - 1]; }
```

Comment:
cuidado com a formatação do código, se for para condicionais curtas pode usar o if em uma linha mas não precisa usar as chaves, fica assim: if (pos > 0) return this.items[pos - 1];